pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+## Requirements
+
+- Filling out the template is required. Any pull request that does not include enough information to be efficiently reviewed may be rejected.
+- Please consider which branch this is to be submitted against. This will normally be the development branch, so please ask your reviewer if you think it needs to go somewhere else. 
+
+## Type
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Improvement to existing feature
+
+## Related JIRA Issue(s)
+
+_Please provide the URL(s) for any JIRA issues related to this PR._
+
+## Description
+
+_Using one or more sentences, describe the proposed changes and the reason for making them._
+
+## Views affected
+
+_List the website view(s) that you know are affected by this change._
+_If possible please provide a relative or localhost URL that can be used to view the change._
+
+### Other effects
+
+_List any other functionality that may be affected or which requires additional changes, such as saved configurations._ <!-- Please add an explanation if, for example, no test is needed -->
+
+- [ ] I have checked any requirements for Google Analytics
```

Comment:
Are we sure we don't want to just create dedicated JIRA tasks for Google Analytics? Without listing the analytics requirements somewhere, I have no idea how to check them (or how the reviewer would verify them). And if the requirements are listed in the JIRA ticket, then what is the point of this checkbox (as opposed to the Description field)?

Reply:
I think it might be quite a complex task to list out everything that needs to be tracked before working on a big feature. It could be up to the developer to decide whether it makes sense to track a particular action and also if it will be useful or not. The idea behind having this checkbox is just to make the developer aware that there is this thing called analytics tracking which could possibly be added/changed/removed to the code changes we made.

Reply:
> It could be up to the developer to decide whether it makes sense to track a particular action and also if it will be useful or not

I am confused. I thought the request to track something should originate with the product owners (Andrea? Steve?) depending on what analytical question they want to have answered. And that therefore it should be decided at the stage of planning a feature. Why would we randomly add analytical events no-one has asked us to?

Reply:
It *should* be considered when planning, but we haven't been doing this routinely to date I think having this final reminder might be useful. We can always remove it later if it doesn't prove to be